### DISCUSSION TITLE
About shared tsconfig.json

### DISCUSSION THREAD

**Discussion by daolou** [rollup/plugins#1208] [Q&A]
![image](https://user-images.githubusercontent.com/22312092/174084655-4c4b3742-3271-4fee-a760-a8b0e98c1b23.png)

### Could you tell me？

**Comment by shellscape** [comment]
They're created with a command such as `ln -s ../../shared/tsconfig.json`. This is a good resource to learn about symlinks https://www.futurelearn.com/info/courses/linux-for-bioinformatics/0/steps/201767#:~:text=A%20symlink%20is%20a%20symbolic,directory%20in%20any%20file%20system.

**Reply by daolou** [reply]
I see.

What I mean is there has automated script instead of executing this command `ln -s ../../shared/tsconfig.json` in every plugin directory.
